<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSlipDlg</class>
 <widget class="QDialog" name="ImportSlipDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Slip</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="QRGroup">
     <property name="title">
      <string>From QR-code</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="GetQRfromClipboardBtn">
        <property name="text">
         <string>Get from clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoadQRfromFileBtn">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GetQRfromCameraBtn">
        <property name="text">
         <string>Get from camera</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="JSONGroup">
     <property name="title">
      <string>From JSON-file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="LoadJSONfromFileBtn">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SlipDataGroup">
     <property name="title">
      <string>From slip data</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="2" column="1">
       <widget class="QLineEdit" name="FD"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="FN"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="FP"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="FNlbl">
        <property name="text">
         <string>FN:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="FPlbl">
        <property name="text">
         <string>FP:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="GetSlipBtn">
        <property name="text">
         <string>Get Slip</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="FDlbl">
        <property name="text">
         <string>FD:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="DummyLbl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="TimestampLbl">
        <property name="text">
         <string>Date/Time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="AmountLbl">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="SlipTimstamp">
        <property name="displayFormat">
         <string>dd/MM/yyyy hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="SlipAmount"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="SlipTypeLbl">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="SlipType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SlipGroup">
     <property name="title">
      <string>Slip</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PeerLbl">
        <property name="text">
         <string>Peer:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LinesLbl">
        <property name="text">
         <string>Lines:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="LoadedLbl">
        <property name="text">
         <string>Imported:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="StoredLbl">
        <property name="text">
         <string>To be added:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="ClearBtn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddOperationBtn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CloseBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
